csharp/test/Microsoft.ML.OnnxRuntime.Tests/InferenceTest.cs

```diff
@@ -226,6 +226,18 @@ public void CanCreateAndDisposeSessionWithModelPath()
                 }
             }
         }
+#if USE_CUDA
+        [Fact]
+        private void TestCUDAProviderOptions()
+        {
+
+            OrtCUDAProviderOptions cuda_options = ProviderOptions.GetDefaultCUDAProviderOptions();
+            using (var sessionOptions = new SessionOptions())
```

Comment:
This does not test anything. Let's find a way to verify that this indeed happened

Reply:
BTW, currently I forgot to pass USE_CUDA in.

Reply:
You can see "Microsoft.ML.OnnxRuntime.Tests.InferenceTest.TestGpu" was skipped in your test run.